components/atlas-uvs/index.html

```diff
@@ -66,7 +66,22 @@ <h1><i>atlas-uvs</i> Component for <a href="https://aframe.io">A-Frame</a></h1>
     <h2>Examples</h2>
 
     <ul id="examples">
-      
+      <li>
+        <a href="examples/static/index-old.html" style="background-color: #222; background-image: url()"></a>
+        <h3>Static with A-Frame 1.1</h3>
+        <p>Simple atlas example</p>
+      </li>
+      <li>
+        <a href="examples/static/index" style="background-color: #222; background-image: url()"></a>
+        <h3>Static with A-Frame 1.2 - Broken right now</h3>
```

Comment:
It says `Broken right now` I imagine we have to remove before merge?

Reply:
removed that text